test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTestStorageAccount.cs

```diff
@@ -0,0 +1,37 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure.Storage;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Import
+{
+    public class ImportTestStorageAccount
```

Comment:
The class is needed as a replacement of CloudStorageAccount class deprecated in v12.